libpod/runtime_pod_infra_linux.go

```diff
@@ -54,7 +54,11 @@ func (r *Runtime) makeInfraContainer(ctx context.Context, p *Pod, imgName, imgID
 	networks := make([]string, 0)
 	options = append(options, WithNetNS(portMappings, isRootless, networks))
 
-	return r.newContainer(ctx, g.Config, options...)
+	ctr, err := r.newContainer(ctx, options...)
+	if err == nil {
+		ctr.SetSpec(g.Config)
```

Comment:
Big no here, we should set the spec only once in newContainer

Reply:
How can we handle the chicken and the egg then unless the spec.  We need the SELinux label reserved within the libpod code.  I don't think this should be reserved outside by the caller.